web/modules/custom/eulogin/joinup_eulogin.module

```diff
@@ -119,6 +132,8 @@ function joinup_eulogin_form_user_form_alter(array &$form, FormStateInterface $f
   if (isset($form['account']['mail']) && \Drupal::config('cas.settings')->get('user_accounts.restrict_email_management')) {
     $form['account']['mail']['#disabled'] = TRUE;
   }
+  $form['field_user_first_name']['widget'][0]['value']['#description'] = t('Your first name is publicly visible.');
+  $form['field_user_family_name']['widget'][0]['value']['#description'] = t('Your last name is publicly visible.');
```

Comment:
I fear that we'll forget to cleanup the code when EU Login will the only login method. Normally, such a hook alter is used to override a default behaviour, which is now in config. But this description is the one that will survive after February 2020. So, instead of overriding here, I'm suggesting that we move this text in config now and override the text only for non-CAS users here. Then we only need to remove these lines in Feb 2020.